### PR TITLE
add support for font size

### PR DESCRIPTION
**Pull request by SIA-WEBSUP**

### CODE REVIEW

**Review by anchovie91471 — APPROVED** [review]
Looks good!

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -83,6 +83,14 @@ function setColorMode(dark) {
                                 </label>
                             </div>
                         @endforeach
+                        <label for="font_size" class="form-label fw-bold mt-4">Font Size</label>
+                        <select name="font_size" id="font_size" class="form-select">
```
Comment (joshreisner) [inline resources/views/home.blade.php:87]: maybe as a follow-up later, use the same laravel syntax for selects here that we use on line 63?

https://github.com/code4recovery/pdf/pull/82/files#diff-866658882d8457254ffdc08e724486f1677695c8e4dd25d6994dd6291b63f2f0R63

Comment (anchovie91471) [reply resources/views/home.blade.php:87]: Thanks @joshreisner! Missed that. @SIA-WEBSUP I can pull and edit the syntax if you prefer.

Comment (SIA-WEBSUP) [reply resources/views/home.blade.php:87]: Please do... I'm away from home for the next few days.
Thank you Both.

File: app/Http/Controllers/Controller.php [hunk 2]
```diff
@@ -40,6 +40,9 @@ public function home()
             'sans-serif' => 'Sans Serif',
             'serif' => 'Serif',
         ];
+
+        $font_sizes = array_combine(range(8, 12), array_map(fn($i) => "{$i} px", range(8, 12)));
```
Comment (joshreisner) [inline app/Http/Controllers/Controller.php:44]: this syntax is new to me, very javascript-y

**Review by joshreisner — APPROVED** [review]

File: app/Http/Controllers/Controller.php (hunk 2 above)
Comment (anchovie91471) [reply app/Http/Controllers/Controller.php:44]: I might need to check that against some older PHP versions now that you mention it.

Comment (joshreisner) [reply app/Http/Controllers/Controller.php:44]: we can control the php version on this service so it should be fine if it's bleeding-edge

Comment (anchovie91471) [reply app/Http/Controllers/Controller.php:44]: Derp. any idea what version we are currently running?

Comment (joshreisner) [reply app/Http/Controllers/Controller.php:44]: pdf is currently on `8.2` but it's easy to change if needed

**Pull request merged**